features/main/custom_normalized.feature

```diff
@@ -50,15 +50,15 @@ Feature: Using custom normalized entity
     """
 
   Scenario: Create a resource with a custom normalized dummy and an id
-    When I add "Content-Type" header equal to "application/json"
+    When I add "Content-Type" header equal to "application/ld+json"
     When I add "Accept" header equal to "application/json"
     And I send a "PUT" request to "/related_normalized_dummies/1" with body:
     """
     {
       "name": "My Dummy",
       "customNormalizedDummy":[{
         "@context": "/contexts/CustomNormalizedDummy",
-        "@id": "/custom_normalized_dummies/1",
+        "@id": "app_dev.php/custom_normalized_dummies/1",
```

Comment:
Why `app_dev.php`?

Reply:
this is specifically a fix for #857 (#877) and i needed a way to force the custom behaviour in the Json LD ItemNormalizer. i.e the iri must be incorrect.